Importers/Rhino/Speckle.Importers.Rhino/Program.cs

```diff
@@ -103,12 +103,12 @@ string token
       try
       {
         var sender = ActivatorUtilities.CreateInstance<Sender>(container);
-        var versionId = await sender.Send(projectId, modelId, new Uri(serverUrl), token);
+        var version = await sender.Send(projectId, modelId, new Uri(serverUrl), token);
```

Comment:
i guess `sender` is another way of operation that we do not use in connectors?

Reply:
Nope, Sender here is in the `Speckle.Importers.Rhino`